config/locales/de.yml

```diff
@@ -1812,6 +1812,7 @@ de:
             placeholder: Standard (%{text})
           text_input_view:
             required_field: Muss ausgefüllt werden
+            max_characters_exceeded: Maximale Zeichenanzahl überschritten (%{max_length})
```

Comment:
Please see https://github.com/codevise/pageflow/wiki/Adding-and-Deleting-Translation-Keys for how to change locales in PRs. `locales/{de,en}.yml` are basically generated files which cannot easily be changed directly.

Reply:
Done.